index.html

```diff
@@ -3121,6 +3127,14 @@ <h3 id="applying">
             prefix.
           </p>
           <p>
+            We encourage implementors to add proprietary extensions to our
+            <a href=
+            "https://github.com/w3c/manifest/wiki/Extensions-Registry">Extensions
+            Registry</a>. This allows the community to track what extension to
+            be made and periodically consider those extensions for
+            standardization.
```

Comment:
> Something like the above would be great as part of the introduction of
> the wiki page.

any hope that people actually read it and put it in as intended, instead 
of just picking the first table and adding it there? if there's 
structure, it would be good to have some model of how it is maintained.